jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/ImageName.java

```diff
@@ -80,16 +83,30 @@ public ImageName(String fullName, String givenTag) {
 
         // set digest to null as default
         digest = null;
+
+        String algo_reg = "@sha(\\d+):";
```

Comment:
This would even match invalid algorithms like `sha100`

Reply:
okay, I will modify this accordingly.